app/parse.py

```diff
@@ -14,5 +20,57 @@ class Course:
     course_type: CourseType
 
 
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(levelname)8s]: %(message)s",
+    handlers=[
+        logging.FileHandler("parser.log"),
+        logging.StreamHandler(sys.stdout),
+    ],
+)
+
+
+def parse_single_course_card(
+        course_soup: BeautifulSoup, course_type: CourseType
+) -> Course:
+    name = course_soup.select_one("a.ProfessionCard_title__Zq5ZY > h3").text
+    short_description = course_soup.select_one(
+        ".ProfessionCard_cardWrapper__JQBNJ > .mb-32"
+    ).text
+
+    logging.info(f"Parsing course {name}, {course_type.value}")
+
+    return Course(
+        name=name, short_description=short_description, course_type=course_type
+    )
+
+
 def get_all_courses() -> list[Course]:
-    pass
+    logging.info("Start parsing courses page")
+
+    page = requests.get(MATE_URL).content
+    soup = BeautifulSoup(page, "html.parser")
+
+    courses = soup.select(".ProfessionCard_cardWrapper__JQBNJ")
+    courses_list = []
```

Comment:
```suggestion
    courses = []
```
_list suffix is redundant